### PR TITLE
Update layouting to use layout management

### LINKED ISSUE #9: Cut-off GUI elements
The quick tracker and Enter are misplaced . They are not shown properly . 
Refer the screenshot .
pizza was tasty

### PR DESCRIPTION
**Pull request by raefaldhia**
+Update to layout management

Fixes #9 

Known Issue:
- [ ] Help Icon
- [ ] Window not showing after clicked `HelpIcon`
- [ ] AddGlue make graybox as shown below
screenshot from 2016-12-26 18-45-48

screenshot from 2016-12-26 18-42-04
screenshot from 2016-12-26 18-42-39

screenshot from 2016-12-26 18-43-12

screenshot from 2016-12-26 18-43-31
screenshot from 2016-12-26 18-46-32

screenshot from 2016-12-26 18-43-49
screenshot from 2016-12-26 18-46-47

screenshot from 2016-12-26 18-44-14
screenshot from 2016-12-26 18-47-13

screenshot from 2016-12-26 18-44-37
screenshot from 2016-12-26 18-47-37

screenshot from 2016-12-26 18-44-55
screenshot from 2016-12-26 18-47-57

screenshot from 2016-12-26 18-45-03
screenshot from 2016-12-26 18-48-07

screenshot from 2016-12-26 18-45-14
screenshot from 2016-12-26 18-48-17

screenshot from 2016-12-26 18-45-32
screenshot from 2016-12-26 18-48-28

screenshot from 2016-12-26 18-45-48
screenshot from 2016-12-26 18-48-40

### CODE REVIEW

File: src/AccountSettingsWindow.cpp [hunk 1]
```diff
@@ -25,70 +26,61 @@ AccountSettingsWindow::AccountSettingsWindow(Account *account)
 	AddShortcut('W',B_COMMAND_KEY,new BMessage(B_QUIT_REQUESTED));
 	AddShortcut('Q',B_COMMAND_KEY,new BMessage(B_QUIT_REQUESTED));
 	
-	BView *back = new BView(Bounds(),"back",B_FOLLOW_ALL,B_WILL_DRAW);
+	BView *back = new BView(NULL, B_WILL_DRAW);
```
Comment (waddlesplash) [inline src/AccountSettingsWindow.cpp]: Keep the name, please.

File: src/CheckNumBox.cpp [hunk 2]
```diff
@@ -122,6 +122,22 @@ CheckNumBox::CheckNumBox(const BRect &frame, const char *name, const char *label
 
 	SetCharacterLimit(8);
 }
+CheckNumBox::CheckNumBox(const char *name, const char *label,
```
Comment (waddlesplash) [inline src/CheckNumBox.cpp]: You can get rid of the non-layout constructors on this and all the other custom classes.

**Comment by waddlesplash** [comment]
There's also a bunch of commented-out code that you can remove.

File: src/AccountSettingsWindow.cpp (hunk 1 above)
Comment (raefaldhia) [inline src/AccountSettingsWindow.cpp]: Ooppss

**Comment by raefaldhia** [comment]
@waddlesplash I'm not sure what comment should i remove. the commentted-out code that i made?
And also could you help me, help button won't shows the help window after click, did i do something wrong in the code?
https://github.com/raefaldhia/CapitalBe/blob/layout-api/src/CheckView.cpp#L58
https://github.com/raefaldhia/CapitalBe/blob/layout-api/src/HelpButton.cpp#L24

**Comment by waddlesplash** [comment]
Very nice, thanks!

**Pull request merged**